src/main/java/org/silsagusi/joonggaemoa/domain/survey/controller/SurveyController.java

```diff
@@ -85,30 +86,45 @@ public ResponseEntity<ApiResponse<Void>> updateSurvey(
 		return ResponseEntity.ok(ApiResponse.ok());
 	}
 
-	@GetMapping("/api/agents/{agentId}/surveys")
+	@GetMapping("/api/surveys")
 	public ResponseEntity<ApiResponse<List<SurveyResponse>>> getAllSurveys() {
 		List<SurveyCommand> surveyCommandList = surveyService.getAllSurveys();
 		List<SurveyResponse> surveyResponseList = surveyCommandList.stream()
 			.map(it -> SurveyResponse.of(it)).toList();
```

Comment:
```suggestion
                        .map(SurveyResponse::of).toList();
```

여기서도 람다식을 메서드 참조로 바꾸면 일관성 있을 거 같아요!